addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Oops - this is a no-go. GPL license is incompatible with the EPL license, so this code cannot be added in the binding here. Why did you decide to add (and refactor) the sources here instead of using the official jar of this lib?
I see two options:
1. Try to get in contact with Dave and see if he is ok to contribute this code under the EPL (and thus become a co-author of this PR).
2. Check with Dave whether he is ok to add a classpath exception (CPE) to his license terms, then we would be able to include a jar of his library within this bundle.